protocol/js/flow-types-chat.js

```diff
@@ -16,6 +16,60 @@ export type RPCError = {
   code: number,
   desc: string
 }
+export type WaitingHandlerType = (waiting: boolean, method: string, sessionID: number) => void
+
+// $FlowIssue we're calling an internal method on engine that's there just for us
+const engineRpcOutgoing = (...args) => engine._rpcOutgoing(...args)
```

Comment:
Yeah, I listed some thoughts on that here:
https://github.com/keybase/client/pull/4060

I think constructing (non-simple things) via import and using import to enforce singleton-ness is worth avoiding.

Also especially in this case because engine construction I think triggers an rpc connection.

Not a huge change to make this happen, we can do it after in a separate PR.